src/field.h

```diff
@@ -352,4 +352,8 @@ static int secp256k1_fe_is_square_var(const secp256k1_fe *a);
 /** Check invariants on a field element (no-op unless VERIFY is enabled). */
 static void secp256k1_fe_verify(const secp256k1_fe *a);
 
+/** Assert a context-specific limit m on the magnitude of a, where
+ *  m must be in the supported range [0,32] (no-op unless VERIFY is enabled). */
+static void secp256k1_fe_verify_magnitude(const secp256k1_fe *a, int m);
```

Comment:
nits in the comment: 
 - s/Assert/Check (Assert is actually the better term, but we write check everywhere else for this. Or change it in the entire code base ;))
 - "context-specific" can be a bit unclear. And I don't think it adds much.
 - Maybe just: "Check that the magnitude of a is at most m. (...) "?

Reply:
Thanks, went with your suggestion.